test/integration/http2_flood_integration_test.cc

```diff
@@ -1076,26 +1105,75 @@ TEST_P(Http2FloodMitigationTest, UpstreamPingFlood) {
   config_helper_.setUpstreamOutboundFramesLimits(AllFrameFloodLimit, ControlFrameFloodLimit);
   initialize();
 
+  floodClient(Http2Frame::makePingFrame(), ControlFrameFloodLimit + 1,
+              "cluster.cluster_0.http2.outbound_control_flood");
+}
+
+TEST_P(Http2FloodMitigationTest, UpstreamSettings) {
+  if (!Runtime::runtimeFeatureEnabled("envoy.reloadable_features.new_codec_behavior")) {
```

Comment:
I think we should make a `bool initializeUpstreamFloodTest()` helper with the boilerplate that goes down to the `initialize()` call as it is repeated in all tests.

Reply:
Done